File: cloud/dao/system_tenant_package.go
package dao

import "github.com/abulo/ratel/v3/stores/null"

// SystemTenantPackage 租户套餐包 system_tenant_package
type SystemTenantPackage struct {
	Id            *int64        `db:"id" json:"id"`                         //bigint 套餐编号,PRI
	Name          *string       `db:"name" json:"name"`                     //varchar 套餐名称
	Status        *int32        `db:"status" json:"status"`                 //tinyint 状态（0正常 1停用）
	SystemMenuIds null.JSON     `db:"system_menu_ids" json:"systemMenuIds"` //json 目录编号
	Remark        null.String   `db:"remark" json:"remark"`                 //varchar 备注
	Deleted       *int32        `db:"deleted" json:"deleted"`               //tinyint 是否删除(0否 1是)
	Creator       null.String   `db:"creator" json:"creator"`               //varchar 创建人
	CreateTime    null.DateTime `db:"create_time" json:"createTime"`        //datetime 创建时间
	Updater       null.String   `db:"updater" json:"updater"`               //varchar 更新人
	UpdateTime    null.DateTime `db:"update_time" json:"updateTime"`        //datetime 更新时间
}

File: admin/src/api/interface/systemTenantPackage.ts
// system_tenant_package 租户套餐包
import { ReqPage } from "./index";
export namespace SystemTenantPackage {
  export interface ReqSystemTenantPackageList extends ReqPage {
    deleted?: number; // 是否删除(0否 1是)
    status?: number; // 状态（0正常 1停用）
    name?: string; // 套餐名称
  }
  export interface ResSystemTenantPackageItem {
    id: number; // 套餐编号
    name: string; // 套餐名称
    status: number; // 状态（0正常 1停用）
    menuIds: number[] | undefined; // 目录编号
    remark: string | undefined; // 备注
    deleted: number; // 是否删除(0否 1是)
    creator: string | undefined; // 创建人
    createTime: string | undefined; // 创建时间
    updater: string | undefined; // 更新人
    updateTime: string | undefined; // 更新时间
  }
}
